website/public/js/controllers/rootCtrl.js

```diff
@@ -184,23 +184,73 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
           $rootScope.charts[id] = (history.length == 0) ? false : !$rootScope.charts[id];
           if (task && task._editing) task._editing = false;
       }
+
+      if($rootScope.charts[id]) {
```

Comment:
I see a lot of cases with spaces between the if and the parenthesis, and also a lot of cases not with. Is there a code styleguide that is being followed?